packages/captp/src/captp.js

```diff
@@ -538,11 +614,9 @@ export const makeCapTP = (
         // Deliver the object, even though we're unplugged.
         rawSend(obj);
       }
-      for (const pr of questions.values()) {
-        pr.rej(reason);
-      }
-      for (const pr of imports.values()) {
-        pr.rej(reason);
+      slotToVal.clear();
+      for (const settler of settlers.values()) {
```

Comment:
Well, unifying `questions` and `imports` does pay off. I suppose I'm less surprised now.

(No change suggested)